cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -937,7 +937,7 @@ def move_base_rel(self, component_name, parameter_name=[0,0,0], blocking=True):
 			ah.set_failed(3, message)
 			return ah
 		max_rel_trans_step = 1.0 # [m]
-		max_rel_rot_step = math.pi/2 # [rad]
+		max_rel_rot_step = math.pi*2+0.1 # [rad]
```

Comment:
can you describe shortly what the problem was without this patch? just for documentation...